Mathlib/Analysis/Normed/Module/Alternating/Uncurry/Fin.lean

```diff
@@ -165,21 +166,80 @@ theorem alternatizeUncurryFin_constOfIsEmptyLIE_comp (f : E →L[𝕜] F) :
   ext
   simp [alternatizeUncurryFin_apply]
 
+@[simp]
+lemma _root_.Fin.removeNth_fun_const {α : Type*} {n : ℕ} (i : Fin (n + 1)) (a : α) :
+    i.removeNth (fun _ ↦ a) = (fun _ ↦ a) :=
+  rfl
+
+/-- If `f` is a continuous bilinear map taking values in the space of continuous alternating maps,
+then evaluation of the twice uncurried `f` on a tuple of vectors `v`
+can be represented as a sum of
+
+$$
+f(v_i, v_j; v_0, \dots, \hat{v_i}, \dots, \hat{v_j}-) -
+f(v_j, v_i; v_0, \dots, \hat{v_i}, \dots, \hat{v_j}-)
+$$
+
+over all `(i j : Fin (n + 2))`, `i < j`, taken with appropriate signs.
+Here $$\hat{v_i}$$ and $$\hat{v_j}$$ mean that these vectors are removed from the tuple.
+
+We use pairs of `i j : Fin (n + 1)`, `i ≤ j`,
+to encode pairs `(i.castSucc : Fin (n + 2), j.succ : Fin (n + 2))`,
+so the power of `-1` is off by one compared to the informal texts.
```

Comment:
Can you say a bit more about the reasoning for this choice? From what I can tell, it seems like the point is to make the statement as clean as possible, which makes sense. But if we're going to differ from the literature, even if only in appearance, it would be nice to have clear justification.

Reply:
Maybe you can disregard this, as I see this is not the only place this choice has been implemented.